torch/nn/quantized/modules/conv.py

```diff
@@ -14,9 +14,48 @@
 
 
 class Conv2d(_ConvNd):
+    r"""Applies a 2D convolution over a quantized input signal composed of
+    several quantized input planes.
+
+    For details on input arguments, parameters, and implementation see
+    :class:`~torch.nn.quantized.Conv2d`.
+
+    .. note::
+        Only `zeros` is supported for the :attr:`padding_mode` argument.
+
+    .. note::
+        Only `torch.quint8` is supported for the input data type.
+
+    .. note::
+        The :attr:`weight` is stored as a packed tensor, and can be accessed
+        using :attr:`_packed_weight` attribute.
```

Comment:
does this mean this module doesn't have `.weight` present? the language doesn't make that clear, or give the user an understanding of why `.weight` is simply not used and a new attribute name is introduced

Reply:
Oh, yea -- this is confusing. The `weight` still exists -- I guess I should remove the `_packed_weight` from the docs.